File: Programing_Basics_with_JS/forLoopMoreExercises/06.Bills.js
function bills(input) {
    let index = 0;
    let numbersOfMonth = Number(input[index]);
    index++;

    let waterBillPerMonth = 20;
    let internetBillPerMonth = 15;
    let totalWaterBill = waterBillPerMonth * numbersOfMonth;
    let totalInternetBill = internetBillPerMonth * numbersOfMonth;
    let totalElectricityBill = 0;
    let totalOtherBill = 0;
    
    for(let i = 1; i <= numbersOfMonth; i++) {
        let electricityBillPerMonth = Number(input[index]);
        index++;
        totalElectricityBill += electricityBillPerMonth;
        let otherBillPerMonth = (waterBillPerMonth + internetBillPerMonth + electricityBillPerMonth) * 1.2;
        totalOtherBill += otherBillPerMonth;
    }

    let averageBillPerMonth = (totalWaterBill + totalInternetBill + totalElectricityBill + totalOtherBill) / numbersOfMonth;

    console.log(`Electricity: ${totalElectricityBill.toFixed(2)} lv`);
    console.log(`Water: ${totalWaterBill.toFixed(2)} lv`);
    console.log(`Internet: ${totalInternetBill.toFixed(2)} lv`);
    console.log(`Other: ${totalOtherBill.toFixed(2)} lv`);
    console.log(`Average: ${averageBillPerMonth.toFixed(2)} lv`);
}

bills(['5', '68.63', '89.25', '132.53', '93.53', '63.22'])
bills(['8', '123.54', '231.54', '140.23', '100', '122.4', '430', '178.52', '64.2'])

File: Programming_Basics_with_Python/for_loop_more_exercises/06._bills.py
mounts = int(input())

water_bill_per_month = 20
water_bill_for_all_months = water_bill_per_month * mounts
internet_bill_per_month = 15
internet_bill_for_all_months = internet_bill_per_month * mounts
other_bill_for_all_months = 0
electricity_bill_for_all_months = 0

for m in range(1, mounts + 1):
    electricity_bill_per_month = float(input())
    electricity_bill_for_all_months += electricity_bill_per_month
    other_bill_for_all_months += (electricity_bill_per_month + water_bill_per_month + internet_bill_per_month) * 1.2

average_bill = (water_bill_for_all_months + internet_bill_for_all_months + electricity_bill_for_all_months + other_bill_for_all_months) / mounts

print(f"Electricity: {electricity_bill_for_all_months:.2f} lv")
print(f"Water: {water_bill_for_all_months:.2f} lv")
print(f"Internet: {internet_bill_for_all_months:.2f} lv")
print(f"Other: {other_bill_for_all_months:.2f} lv")
print(f"Average: {average_bill:.2f} lv")
